src/components/converter/ComboInput.js

```diff
@@ -115,37 +135,60 @@ const Input = styled.input`
 `
 
 const Dropdown = styled.div`
-  position; relative;
-  width: 20%;
-  min-width: 114px;
+  position: absolute;
+  right: 0;
+  z-index: 2;
+  width: 120px;
   height: 50px;
-  background: #ffffff;
+  background: transparent;
   display: flex;
   flex-direction: row;
   justify-content: center;
   align-items: center;
-  border-width: 1px 1px 1px 0;
+  overflow: hidden;
+  border-width: 0 0 0 1px;
+  border-radius: 0 4px 4px 0;
   border-style: solid;
   border-color: #dde4e9;
   color: #212b36;
-  border-radius: 0 4px 4px 0;
-  .selected {
-    cursor: pointer;
+  cursor: pointer;
+  .header {
+    position: absolute;
+    left: 5%;
+    padding: 0;
+    width: 100%;
   }
 `
+
 const DropdownContent = styled.div`
   position: absolute;
+  z-index: 100;
```

Comment:
I don’t think we need such a high `z-index`, since we are in the context of `DropDown` which has a `position: absolute`.